src/pandapipes/component_models/abstract_models/circulation_pump.py

```diff
@@ -34,13 +33,23 @@ def get_component_input(cls):
     def get_result_table(cls, net):
         """
 
+        Gets the result table.
+
         :param net: The pandapipes network
         :type net: pandapipesNet
         :return: (columns, all_float) - the column names and whether they are all float type. Only
                 if False, returns columns as tuples also specifying the dtypes
         :rtype: (list, bool)
         """
-        return ["mdot_flow_kg_per_s", "deltap_bar"], True
+        if get_fluid(net).is_gas:
```

Comment:
Can we generally remove velocity results here? Would make much more sense, since we don't need a diameter for simulation anymore.

Reply:
Removed them in most wo internal components except from valves. Velocities in valves seem to be relevant as most tools keep this information.